### PR TITLE
Install all applications as console scripts via setup.py.

### PR DESCRIPTION
**Pull request by adamshapiro0**
This should install them correctly in Windows and Mac, not just Linux. That way users can run:
```
> p1_display path/to/log
```
at the command line in all operating systems.

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -64,6 +57,14 @@ FusionEngine message specification.
           - This class is used by the `bin/p1_display` application
         - [file_reader.py](analysis/file_reader.py) - `FileReader` class, capable of loading and time-aligning
           FusionEngine data captured in a `*.p1log` file
+      - `applications` - Applications and processing tools
+        - [p1_display](bin/p1_display) - Generate HTML plots of vehicle trajectory, etc. (runs `analyzer.py`)
+        - [p1_extract](bin/p1_extract) - Extract FusionEngine message contents from a binary file containing mixed data
+          (e.g., interleaved RTCM and FusionEngine messages)
+        - [p1_lband_extract](bin/p1_lband_extract) - Extract L-band data bits contained in a log of FusionEngine
```
Comment (anathan) [inline python/README.md:64]: isnt this in applications now?

File: python/README.md [hunk 2]
```diff
@@ -64,6 +57,14 @@ FusionEngine message specification.
...
+          `LBandFrameMessage`
+        - [p1_print](bin/p1_print) - Print the contents of FusionEngine messages found in a binary file to the
```
Comment (anathan) [inline python/README.md:66]: same.

File: python/README.md [hunk 3]
```diff
@@ -97,12 +97,13 @@ FusionEngine message specification.
    ```
 3. Install the `fusione-engine-client` module, including all analysis and data processing tools:
    ```bash
-   python3 -m pip install -e /path/to/fusion-engine-client[all]
+   python3 -m pip install -e /path/to/fusion-engine-client
```
Comment (anathan) [inline python/README.md:100]: yes!

**Review by anathan — APPROVED** [review]
just check readme comment, otherwise g2g.

File: python/README.md (hunk 1 above)
Comment (adamshapiro0) [reply python/README.md]: Good catch, fixed and reorganized this stuff a little better

**Pull request merged**